protocol/app/app.go

```diff
@@ -1034,6 +1061,7 @@ func New(
 		ibcexported.ModuleName,
 		ibctransfertypes.ModuleName,
 		ratelimitmoduletypes.ModuleName,
+		icatypes.ModuleName,
```

Comment:
Not really, since the [ICA module](https://github.com/cosmos/ibc-go/tree/main/modules/apps/27-interchain-accounts/host) does not have `EndBlocker` implementation